File: src/main/java/org/example/bankramenserver/domain/recurring/exception/DuplicateRecurringPaymentException.java
package org.example.bankramenserver.domain.recurring.exception;

import org.example.bankramenserver.global.error.exception.ErrorCode;
import org.example.bankramenserver.global.error.exception.GlobalException;

public class DuplicateRecurringPaymentException extends GlobalException {

    public DuplicateRecurringPaymentException() {
        super(ErrorCode.DUPLICATE_RECURRING_PAYMENT);
    }
}

File: src/main/kotlin/org/example/bankramenserver/domain/recurring/exception/DuplicateRecurringPaymentException.kt
package org.example.bankramenserver.domain.recurring.exception

import org.example.bankramenserver.global.error.exception.ErrorCode
import org.example.bankramenserver.global.error.exception.GlobalException

class DuplicateRecurringPaymentException : GlobalException(ErrorCode.DUPLICATE_RECURRING_PAYMENT)
